Add ability to choose a random color of selling shop items (#13634)
OpenRCT2/OpenRCT2: distribution/changelog.txt

- Feature: [#13593] [Plugin] Add ability to read and change the position of ride vehicles.
- Feature: [#13613] Add single-rail roller coaster (Rocky Mountain Construction Raptor).
- Feature: [#13614] Add terrain surfaces from RollerCoaster Tycoon 1.

> - Feature: [#13634] Add ability to choose a random color of the selling shop items.

Reviewer: ```suggestion
- Feature: [#13634] Add ability to sell merchandise in random colours.
```
